amsoell/foteaux
# Dockerize the development environment

Enable the application to run with a Docker environment, including MySQL, MailHog and Minio S3 service support, vi Laravel Sail.

## Test Coverage

_none_

## Acceptance Testing Steps

1. Initialize the docker containers:

```
cp .env.example .env
composer install
./vendor/bin/sail up -d
./vendor/bin/sail artisan key:generate
./vendor/bin/sail artisan migrate
./vendor/bin/sail composer install
./vendor/bin/sail npm install
./vendor/bin/sail npm run dev
```
2. Add the `minio` hostname to your local DNS pointing to `localhost`
3. Navigate to `http://minio:8900/buckets` and create a new bucket called `foteaux`
4. Confirm the development environment is accessible at `http://0.0.0.0`
